<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderCard</class>
 <widget class="QWidget" name="OrderCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="orderIdLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Order ID: 12345</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="userIdLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>User ID: 6789</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="paymentTypeLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Payment Type: Credit Card</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="priceLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Price: $100</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="addressLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Address: 123 Main St</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="phoneLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Phone: (123) 456-7890</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="emailLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Email: example@example.com</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="detailsBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Детали заказа</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
